task/sast-snyk-check/0.1/sast-snyk-check.yaml

```diff
@@ -9,16 +9,16 @@ metadata:
   name: sast-snyk-check
 spec:
   description: >-
-    Static code security test with snyk
+    Static code security test with snyk.
```

Comment:
[this](https://snyk.io/) "snyk"? if so, please change to:

Static application security testing with Snyk.

or is it _for_ Snyk?